core/classes/Core/TimeAgo.php

```diff
@@ -59,7 +59,7 @@ public function inWords($past, Language $language): string {
             $timeDifference < (($this->_secondsPerMinute * 89) + 29)
         ) {
             // between 44mins30secs and 1hour29mins29secs
```

Comment:
Between roughly 45 minutes and 90 minutes should say "about 1 hour", not "more than an hour". The previous strings were correct. Same goes for other strings you changed.